app/kubernetes/services/applicationService.js

```diff
@@ -377,17 +410,12 @@ class KubernetesApplicationService {
     }
 
     if (application.ServiceType) {
```

Comment:
is application.ServiceType still usefull? as an app could have multiple services.